README.md

```diff
@@ -1,4 +1,750 @@
+	func deleteTopic(serviceAPI *adminrestv1.AdminrestV1) error {
+		// Construct an instance of the DeleteTopicOptions model
+		deleteTopicOptionsModel := new(adminrestv1.DeleteTopicOptions)
+		deleteTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+		deleteTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+	
+		// Delete Topic
+		response, operationErr := serviceAPI.DeleteTopic(deleteTopicOptionsModel)
+		if operationErr != nil {
+			return fmt.Errorf("Error Deleting Topic: %s\n", operationErr.Error())
+		}
+	
+		// Check the result.
+		if response.StatusCode != http.StatusAccepted {
+			return fmt.Errorf("Error Deleting Topic: status %d\n", response.StatusCode)
+		}
+	
+		fmt.Printf("\tname: %s deleted\n", *deleteTopicOptionsModel.TopicName)
+		return nil
+	} // func.end
+
+
+
+### Listing Kafka topics
+---
+You can list all of your Kafka topics by issuing a GET request to the
+`/admin/topics` path. 
+
+Expected status codes:
+- 200: the topic list is returned as JSON in the following format:
+```json
+[
+  {
+    "name": "topic1",
+    "partitions": 1,
+    "retentionMs": 86400000,
+    "cleanupPolicy": "delete"
+  },
+  { "name": "topic2",
+    "partitions": 2,
+    "retentionMs": 86400000,
+    "cleanupPolicy": "delete"
+  }
+]
+```
+
+A successful response will have HTTP status code 200 (OK) and contain an
+array of JSON objects, where each object represents a Kafka topic and has the
+following properties:
+
+| Property name     | Description                                             |
+|-------------------|---------------------------------------------------------|
+| name              | The name of the Kafka topic.                            |
+| partitions        | The number of partitions of the Kafka topic.            |
+| retentionsMs      | The retention period for messages on the topic (in ms). |
+| cleanupPolicy     | The cleanup policy of the Kafka topic.                  |
+
+#### Example
+
+	func listTopics(serviceAPI *adminrestv1.AdminrestV1) error {
+		// Construct an instance of the ListTopicsOptions model
+		listTopicsOptionsModel := new(adminrestv1.ListTopicsOptions)
+	
+		// Call ListTopics.
+		result, response, operationErr := serviceAPI.ListTopics(listTopicsOptionsModel)
+		if operationErr != nil {
+			return fmt.Errorf("Error Listing Topics" + operationErr.Error())
+		}
+	
+		// Check the result.
+		if response.StatusCode != http.StatusOK {
+			return fmt.Errorf("Error Listing Topics: status %d\n", response.StatusCode)
+		}
+	
+		// Loop and print topics.
+		for _, topicDetail := range result {
+			fmt.Printf("\tname: %s\n", *topicDetail.Name)
+		}
+		return nil
+	} // func.end
+
+
+
+### Getting a Kafka topic
+---
+To get a Kafka topic detail information, issue a GET request to the `/admin/topics/TOPICNAME`
+path (where `TOPICNAME` is the name of the Kafka topic that you want to get).  
+
+Expected status codes
+  - 200: Retrieve topic details successfully in following format:
```

Comment:
isn't there other codes?

Reply:
Added
- 403: Not authorized.
- 404: Topic does not exist.